samples/build-cache/app/pom.xml

```diff
@@ -1,11 +1,10 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0"
-  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <parent>
     <groupId>io.fabric8.dmp.samples.build-cache</groupId>
     <artifactId>parent</artifactId>
-    <version>0.39.1</version>
+    <version>0.40.0-SNAPSHOT</version>
```

Comment:
In my opinion, samples should be pointing to the latest released versions (unless they contain functionality that hasn't been released yet).

Reply:
What is the workflow of the person using the samples?  Do you expect them to pull this repository and run the sample locally?  What advantage is there to run against released version vs. in-development version?

I think the integration tests should definitely run against in-development version.  I can add a pluginManagement section to the samples aggregator pom to set the version to the released version;  this will allow sharing of "parent" pom.

Reply:
yes, integration tests should definitely run against in-development version. I was only referring to samples/ folder

Reply:
Added pluginManagement section to samples aggregator to use the released version